config/config/wc-config.yaml

```diff
@@ -27,6 +27,15 @@ user:
 ## Falco configuration.
 falco:
   enabled: true
+  customRules: {}
```

Comment:
Can we link to these docs to make writing custom rules more inviting? https://falco.org/docs/rules/